online-store/domain/src/main/java/com/issoft/auto/domain/db/DBManager.java

```diff
@@ -73,7 +74,7 @@ public static void createCategoryTableForDB(){
     public static void createProductTableForDB(){
         try {
 
-            Connection connection = ConnectionUtil.getConnection(url, user, password);
+            Connection connection = ConnectionUtil.getConnection(PropertiesUtil.get(URL), PropertiesUtil.get(USER), PropertiesUtil.get(PASSWORD));
```

Comment:
there no need to specify url and so on, each time, that props could be read in connection utils